mptt/admin.py

```diff
@@ -172,6 +175,13 @@ def changelist_view(self, request, *args, **kwargs):
 
         return response
 
+    def get_data_before_update(self, request, cut_item, pasted_on):
+        return {k: getattr(cut_item, k) for k in ("lft", "mptt_level", "rght", "tree_id")}
```

Comment:
The fields' names aren't necessarily `lft`, `rght` etc. -- they can be overridden unfortunately.

You should inspect `cut_item._mptt_meta`. Also, `mptt_level` isn't even the default name.

Reply:
Really sorry, I completely missed this. We'll correct it and re-submit it.

Reply:
Don't worry, that's exactly what unittests and reviews are for :) 

Thanks!